DuplicateCodeDetector/CloneGroups.cs

```diff
@@ -36,7 +36,10 @@ public CloneGroups(IEnumerable<(string File1, string File2)> clonePairs)
 
             var duplicationFactors = CloneSets.Select(c => c.Count).ToList();
 
-            Console.WriteLine($"Avg Duplication Factor: {duplicationFactors.Average()}");
+            if (duplicationFactors.Count > 0)
+            {
+                Console.WriteLine($"Avg Duplication Factor: {duplicationFactors.Average()}");
+            }
             duplicationFactors.Sort();
             double median;
             int midpoint = duplicationFactors.Count / 2;
```

Comment:
@villelaitila Thanks for the edit above. I see where it could fail when no duplicates are found. 
* I _think_ that the `median` computation should also be in guarded by the `duplicationFactors.Count > 0` Am I missing something?
* Could you add an explicit `else` block that will print a message of the form "No Duplicates Found."?

Thanks :)

Reply:
Yes, even though `median` computation succeeds without crash when there are no duplicates, it does not make sense to do it. I'll update the PR, and also add the `else` block.

Reply:
Now amended the commit and updated here.